charts/applicationset-progressive-sync/templates/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
             {{- if or (gt ( .Values.replicaCount | int64) 1) .Values.args.enableLeaderElection }}
             - --enable-leader-election=true
             {{- end }}
+            - --namespace={{ .Values.args.Namespace }}
+            - --argo-namespace={{ .Values.args.argoNamespace }}
```

Comment:
```suggestion
            - --argo-namespace={{ .Values.args.argoNamespace | defaults .Release.Namespace }}
```

Reply:
Well at this point shouldn't just be `{{ .Release.Namespace }}`?

Reply:
Pushed